testsuite/performance/infinispan/pom.xml

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0"?>
+<!--
+~ Copyright 2016 Red Hat, Inc. and/or its affiliates
+~ and other contributors as indicated by the @author tags.
+~
+~ Licensed under the Apache License, Version 2.0 (the "License");
+~ you may not use this file except in compliance with the License.
+~ You may obtain a copy of the License at
+~
+~ http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the License for the specific language governing permissions and
+~ limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <groupId>org.keycloak.testsuite</groupId>
+        <artifactId>performance</artifactId>
+        <version>4.0.0.CR1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>performance-keycloak-infinispan-server</artifactId>
+    <name>Keycloak Performance TestSuite - Infinispan Server</name>
+    <packaging>pom</packaging>
+    
+    <properties>
+        <infinispan.groupId>org.infinispan.server</infinispan.groupId>
+        <infinispan.artifactId>infinispan-server-build</infinispan.artifactId>
+        <infinispan.version>8.2.8.Final</infinispan.version>
```

Comment:
Can this use ${infinispan.version} from https://github.com/keycloak/keycloak/blob/master/pom.xml#L68?